presto-druid/src/test/resources/broker.config

```diff
@@ -10,7 +10,7 @@ druid.broker.http.maxQueuedBytes=5000000
 
 # Processing threads and buffers
 druid.processing.buffer.sizeBytes=50000000
-druid.processing.numMergeBuffers=1
+druid.processing.numMergeBuffers=2
```

Comment:
Are druid config changes related?

i would assume they aren't, especially since #4313 is outstanding still.

Reply:
Not related until #4313 is merged. removed for now.